queries/content/bech32_address.yml

```diff
@@ -0,0 +1,8 @@
+name: "Cryptocurrency address: Bech32"
+type: "query"
+source: |
+  any([body.plain.raw, body.html.raw], regex.match(., '\b(bc(0([ac-hj-np-z02-9]{39}|[ac-hj-np-z02-9]{59})|1[ac-hj-np-z02-9]{8,87}))\b'))

```

Comment:
WDYT about switching these to `body.html.display_text` or `body.html.inner_text`? raw HTML can be pretty large to sift through

Reply:
Great idea, will update these to use body.html.display_text

Reply:
@rw-access updated